src/app/validators/nation-validator.service.ts

```diff
@@ -15,9 +15,9 @@ export class NationValidatorService {
   readonly dbName = 'nations';
 
   constructor(private couchService: CouchService) {}
-    public nationCheckerService$(name: string): Observable<boolean> {
+    public nationCheckerService$(name: string, nationUrl: string): Observable<boolean> {
```

Comment:
`nationCheckerService` is only passed one `AbstractControl` at a time, so it will not receive both the `name` and `nationUrl`.